.travis.yml

```diff
@@ -43,6 +43,11 @@ before_script:
       dbus-launch /usr/bin/python -c \
         "import gnomekeyring;gnomekeyring.create_sync('login', '');"
     fi
+  - npm install -g node-gyp@3.7.0
+  - |
+    if [[ "$TRAVIS_OS_NAME" == "linux" ]]; then
+      node-gyp configure -- -Dbuild_v8_with_gn=false
```

Comment:
I think I'm officially stumped as to how to set this correctly.

@daviwil @vanessayuenn any ideas aside from [this workaround](https://github.com/daviwil/node-pty-prebuilt/commit/25aebf8ddd711bedb55bedb360578274fcb1dff0)?

Reply:
i backfilled the variable in `binding.gyp` instead so at least CI stopped complaining about lack of `build_v8_with_gn`, but now it's failing at `/usr/include//c++/4.8/cstdio:120:11: error: no member named 'gets' in the global namespace`.